### PR TITLE
get rid of Python 2.4 and 2.5 builders

### PR DESCRIPTION
**Pull request by sa2ajj**
* minimum requirement for both master and build slave is Python 2.6

### CODE REVIEW

**Comment by djmitche** [comment]
looks good

File: slaves.py [hunk 1]
```diff
@@ -97,7 +96,6 @@ def __init__(self, name, **kwargs):
             run_config=True,
             tw0810=True,
             py24=True,
```
Comment (seankelly) [inline slaves.py]: A `py24=True` here but assuming don't want this anymore.

File: slaves.py [hunk 2]
```diff
@@ -26,7 +26,6 @@ class MySlaveBase(object):
     # are given, too
     run_config = False
     py24 = False
```
Comment (seankelly) [inline slaves.py]: Might as well delete the `py24` bit too.

File: slaves.py [hunk 3]
```diff
@@ -108,7 +106,6 @@ def __init__(self, name, **kwargs):
             run_single=False,
             run_config=True,
             py24=False,
```
Comment (seankelly) [inline slaves.py]: Might as well delete the `py24` bit too.

File: builders.py [hunk 4]
```diff
@@ -87,9 +87,7 @@ def start(self):
         echo "Checking for simplejson or json";
         "$VEPYTHON" -c 'import json' 2>/dev/null || "$VEPYTHON" -c 'import simplejson' ||
                     "$VE/bin/pip" install --no-index --download-cache="$PWD/.." --find-links="$PKG_URL" simplejson || exit 1;
-        echo "Checking for sqlite3, including pysqlite3 on Python 2.5";
         "$VEPYTHON" -c 'import sqlite3, sys; assert sys.version_info >= (2,6)' 2>/dev/null ||
-                    "$VEPYTHON" -c 'import pysqlite2.dbapi2' ||
                     "$VE/bin/pip" install --no-index --download-cache="$PWD/.." --find-links="$PKG_URL" pysqlite || exit 1
```
Comment (seankelly) [inline builders.py]: If I'm reading this shell right, it: checks if the python version is 2.6 or higher, if that fails it tries to import `pysqlite2.dbapi2`, if that fails then it installs via pip. The first check is no longer going to fail now so the import check and pip install lines can be removed.

Comment (seankelly) [inline builders.py]: If old lines 92 and 93 are removed then line 91 is only asserting that the python version is at least 2.6 and that'll not fail too. So, old lines 91 and 93 can be removed.

Comment (sa2ajj) [inline builders.py]: Good catch, thank you

**Pull request merged**